Rakefile

```diff
@@ -22,3 +22,21 @@ task tag: :build do
   system "git tag -a v#{ Measured::VERSION } -m 'Tagging #{ Measured::VERSION }'"
   system "git push --tags"
 end
+
+task :environment do
+  require 'measured'
+end
+
+namespace :cache do
+  task write: :environment do
+    class Measured::Cache::Json
+      prepend Measured::Cache::JsonWriter
```

Comment:
This extracts the `#write` method into a module that is never loaded in the application. If we did a cache write from in the app it would overwrite the JSON file in the currently installed gem. You never want this to happen. So here we only make it available from the rake task that regenerates the file.